app/services/albums.js

```diff
@@ -0,0 +1,40 @@
+const request = require('request-promise');
+
+const errors = require('../errors');
+const logger = require('../logger');
+const { albumsApi } = require('../../config').common;
+
+exports.getAlbum = id => {
+  const options = {
+    method: 'GET',
+    uri: `${albumsApi.endpoint}${albumsApi.routes.albums}/${id}`,
+    json: true
+  };
+  logger.info(`Request to make: ${options.method} ${options.uri}`);
+
+  return request(options)
+    .then(album =>
+      this.getPhotos({ albumId: album.id }).then(photos => ({
```

Comment:
use exports rather than this, we try to avoid the usage, otherwise its fine

Reply:
Ahh ok. Can I change it in the next PR? I have to do similar corrections on that one anyway